ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -81,7 +87,10 @@ def wf(ctx: task.OrchestrationContext, inp: TInput):
         else:
             # this case should ideally never happen
             wf.__name__ = workflow.__name__
-        return self.__obj.call_sub_orchestrator(wf, input=input, instance_id=instance_id)
+        if retry_policy is None:
+            return self.__obj.call_sub_orchestrator(wf, input=input, instance_id=instance_id)
+        return self.__obj.call_sub_orchestrator(wf, input=input, instance_id=instance_id,
```

Comment:
please add tests for this line and the previous call_activity also.

Reply:
Test for call_activity retry was already included in demo_workflow. 
I have also included child_workflow retries and retries for activity inside child_workflow.